### PR TITLE
Added StakeEase

### PR DESCRIPTION
**Pull request by vatsalgupta13**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
StakeEase

##### Twitter Link:
https://x.com/StakeEase

##### List of audit links if any:
https://docs.stakeease.com/stakeease/security/audit-reports

##### Website Link:
https://stakeease.com/

##### Logo (High resolution, will be shown with rounded borders):
https://2042496576-files.gitbook.io/~/files/v0/b/gitbook-x-prod.appspot.com/o/spaces%2FbDyh5QHxaiQdaNz8Ud6D%2Fuploads%2FibXBrcqBoB8MM1Ni99i2%2FIcon%20on%20Black%20BG.png?alt=media&token=42e963d1-b2fc-4583-96ec-e0739cbb972f

##### Current TVL:
$927000

##### Chain:
Ethereum (1) and Taiko (167000)

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): 
(https://api.coingecko.com/api/v3/coins/list)
CL1510240030

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
995893

##### Short Description (to be shown on DefiLlama):
StakeEase is a restaking index platform designed to unify the fragmented liquid restaking landscape. By consolidating multiple LRTs into a single vault, StakeEase simplifies the restaking experience for the user, gives them access to diversified rewards, opens up new yield-generating avenues, and reduces LRT fragmentation in the L2s.

##### Token address and ticker if any:
Address (Ethereum): 0x3Ae86d2d2f6A8d29792156Dd92DB01D7B3CC75Fb 
Ticker: sxETH

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Liquid Restaking

##### forkedFrom (Does your project originate from another project):
Origin ETH: https://docs.originprotocol.com/protocol/oeth

##### methodology (what is being counted as tvl, how is tvl being calculated):
StakeEase vault consists of rsETH, ezETH, rswETH, eETH and WETH tokens. The TVL being calculated is the sum of the token balances of the StakeEase vault. 

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/stakeease

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/stakeease exports TVL: 
        
 
 
    ethereum                  45.96 k
    
    total                    45.96 k

**Comment by llamatester** [comment]
The adapter at projects/stakeease exports TVL: 
        
 
 
    ethereum                  927.74 k
    
    total                    927.74 k

File: projects/stakeease/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    })
+    api.add(WETH_TOKEN_ADDRESS, WETH_BALANCE)
+    api.add(eETH_TOKEN_ADDRESS, eETH_BALANCE)
+    api.add(ezETH_TOKEN_ADDRESS, ezETH_BALANCE)
+    api.add(rswETH_TOKEN_ADDRESS, rswETH_BALANCE)
+    api.add(rsETH_TOKEN_ADDRESS, rsETH_BALANCE)
+}
+
+module.exports = {
+    doublecounted: false,
```
Comment (Define101) [inline projects/stakeease/index.js:27]: doublecounted: true, 
not false

**Comment by llamatester** [comment]
The adapter at projects/stakeease exports TVL: 
        
 
 
    ethereum                  1.56 M
    
    total                    1.56 M

File: projects/stakeease/index.js (hunk 1 above)
Comment (vatsalgupta13) [reply projects/stakeease/index.js]: Made the change @Define101. Could you kindly merge this PR?

**Comment by g1nt0ki** [comment]
thanks for the PR

**Comment by vatsalgupta13** [comment]
Seems like this has not been merged. Is there any issue here?

**Comment by g1nt0ki** [comment]
merged here: https://github.com/DefiLlama/DefiLlama-Adapters/commit/9f6e65563ab9992a03354ce24b6fe476b7751387

**Pull request closed without merging**